make-kamikaze.sh

```diff
@@ -21,6 +21,7 @@ setup_port_forwarding
 install_dependencies
 create_user
 install_redeem
+virtualize_redeem
```

Comment:
Beyond the ability to have multiple instances of Redeem installed in parallel, what are the other benefits to virtualizing Redeem in venv? I know we've had the discussion but I forgot the other positive aspects of the virtualization.

Reply:
Best practices are to run individual services each in their own "container". Virtual environments are the local version of a "container" for python. Keeping things in separate containers is extremely important if the user can break "packaging" and update part of the system without doing the rest.

Reply:
Thanks - and agreed, making it harder to break things is good.